addons/canal/canal.yaml

```diff
@@ -411,7 +411,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: '{{ Registry "docker.io" }}/calico/cni:v3.8.0'
+          image: '{{ Registry "quay.io" }}/calico/cni:v3.8.0'
```

Comment:
v3.8.0 (as well as other 3.8 versions) is missing multi-arch manifest (so would not work on ARM nodes).
Let's keep it on dokerhub.